test/extended/apiserver/api_requests.go

```diff
@@ -85,8 +85,7 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 		sort.Strings(failureOutput)
 
 		if len(failureOutput) > 0 {
-			// don't insta-fail all of CI
-			result.Flakef(strings.Join(failureOutput, "\n"))
```

Comment:
[Unit test results][1]:

```
# github.com/openshift/origin/test/extended/apiserver
test/extended/apiserver/api_requests.go:18:2: imported and not used: "github.com/openshift/origin/pkg/test/ginkgo/result"
```

Can we drop the import, now that you're removing the only consumer here?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27561/pull-ci-openshift-origin-master-unit/1594661183284056064#1:build-log.txt%3A23